test/manual-scenarios/swap/swap eth.csv

```diff
@@ -5,8 +5,8 @@ Step,Test steps,Preconditions,Test data,Expected result,Notes
 4,"Click ""Fork a network"". Leave network field as default Mainnet, click ""Create"".",,,,
 5,"Click ""Add RPC to MetaMask"". Proceed to add new network and switch to new network on MetaMask Dialog screen.",,,"The selected network is the fork of Ethereum Mainnet started in previous step.  The Ether balance is shown on the overview. The wallet address is shown on the overview. ",Take a note of the wallet address.
 6,"Go back to Tenderly, click ""Fund Accounts"". Enter wallet address we copied in previous step and click ""Submit"".",,,Wait a bit and see the balance on MeteMask has been funded with more Ether.,
-7,"Click ""Swap"" button on MetaMask home screen.",,,Swap screen is shown. Default swap from currency is ETH. Settings button is shown.,
-8,Click on settings button on swap screen. Make sure smart swap in settings is set to OFF.,,,"Slippage tolerance section is displayed. User can choose between 2%, 3%, or enter a custom value.",
+7.,Go to Settings. Make sure Smart Transactions in settings is set to OFF.,,,
+8,"Click ""Swap"" button on MetaMask home screen.",,,"Swap screen is shown. Default swap from currency is ETH. Settings button is shown. Slippage tolerance section is displayed. User can choose between 2%, 3%  or enter a custom value.",,,
 9,"Close the settings modal. On swap screen, enter a value for ETH that user wants to swap, then select a token to swap to.",,"e.g.  0.1 ETH to swap, and choose DAI to swap to",The amount for selected token user will obtain is displayed. Swap quote rate is shown. Estimated gas fee and max fee are shown. Included MetaMask fee is shown.,Take a note for the amount of selected token that user will obtain.
```

Comment:
I looks like there is a formatting issue in this file, as I cannot see the table previewed like in the other files

![Screenshot from 2024-05-22 17-05-26](https://github.com/MetaMask/metamask-extension/assets/54408225/626b5c8c-e524-4e2c-afe4-4186addf3189)
- :red_circle:  this file https://github.com/MetaMask/metamask-extension/blob/modify-smart-swap-manual-scenario/test/manual-scenarios/swap/swap%20eth.csv
- :green_circle:  smart tx file https://github.com/MetaMask/metamask-extension/blob/modify-smart-swap-manual-scenario/test/manual-scenarios/swap/smart%20transactions.csv

Reply:
@seaona thank you for the observation, can you please check if it's okay now? It works on my side after the latest change.

Reply:
It looks good now, thank you!